lvgl_touch/touch_driver.c

```diff
@@ -40,6 +42,8 @@ bool touch_driver_read(lv_indev_drv_t *drv, lv_indev_data_t *data)
     res = FT81x_read(drv, data);
 #elif defined (CONFIG_LV_TOUCH_CONTROLLER_RA8875)
     res = ra8875_touch_read(drv, data);
+#elif defined (CONFIG_LV_TOUCH_CONTROLLER_GT911)
+    res = gt911_read(drv, data);
 #endif
 
     return res;
```

Comment:
Note that this needs an update for v8 as the return value is ignored there.

You can do it without adjusting the drivers with this change:

```suggestion
#if LVGL_VERSION_MAJOR < 8
    data->continue_reading = res;
#else
    return res;
#endif
```

You would also have to conditionally change the prototype of `touch_driver_read` to return `void`.

Reply:
Thanks for the information @embeddedt , I think the `#if LVGL_VERSION_MAJOR < 8` must be `#if LVGL_VERSION_MAJOR >= 8` instead, isn't?

Reply:
Yes; I mixed up the two versions. Sorry.

I think you only need to update `touch_driver.c` so you can probably keep that change in this PR.

Reply:
@embeddedt is this change what you meant?

Reply:
Yes; both changes are perfect.

Reply:
Yes; both changes are perfect.